azure-spring-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/SpringConfiguration.java

```diff
@@ -10,20 +10,15 @@
 
 public class SpringConfiguration {
 
-    private int port;
-    private boolean isPublic;
+    private Boolean isPublic;
     private String subscriptionId;
     private String resourceGroup;
```

Comment:
remove this?

Reply:
We could remove it if we list cluster directly without resource group, this will be done in later pr.